### DISCUSSION TITLE
articles/santoka-haiku

### DISCUSSION THREAD

**Discussion by giscus** [hutusi/hutusi.github.com#162] [Comments]
# articles/santoka-haiku

在微信读书上读了本《日本短诗套装》，其实是几本书的一个合集，而这几本书也是对日本一些俳句和和歌诗人的集录。分别是松尾芭蕉俳句集《但愿呼我为旅人》、小林一茶俳句集《这世界如露水般短暂》、与谢芜村俳句集《春之海终日悠哉悠哉》、种田山头火俳句集《只余剩米慢慢煮》1以及日本短歌杂集《夕颜》。除了种田山头火外，其他四本都是由台湾人陈黎和张芬龄翻译。我对这两位台湾人的翻译水平真是一言难尽，完全感受不到诗歌的味道。而且胡乱用词，且不说雅达，连信都做不到。什么“梦工场”、“粉丝”、“黑胶片”、“唱盘”、“冬季放屁奥运会”，这些现代的词语放在古代的俳句词人口中，怎么读也不通顺呀。到豆瓣上一搜，发现大家都在骂译者，我这才恍然大悟，上当了，不该看这二人的译本，白白浪费了许多时间。 《只余剩米慢慢煮》 ↩

https://hutusi.com/articles/santoka-haiku

**Comment by semyin** [comment]
> 当时就想起了小时候妈妈做布鞋，按鞋底的形状剪报纸，再用米糊将报纸一层层粘起来，中间夹杂着些破布，做成鞋底。往事便历历在目，今天正是母亲的十三年忌日，难忘怀。

以前上学的时候，特别喜欢穿妈妈纳的布鞋，好几双鞋都被大脚趾都戳出洞，补一补又能穿几个月。干净，透气

**Reply by hutusi** [reply]
满满的回忆呀！
